pandas/core/categorical.py

```diff
@@ -1157,30 +1157,74 @@ def argsort(self, ascending=True, **kwargs):
         return result
 
     def sort_values(self, inplace=False, ascending=True, na_position='last'):
-        """ Sorts the Category by category value returning a new Categorical by
-        default.
-
-        Only ordered Categoricals can be sorted!
-
-        Categorical.sort is the equivalent but sorts the Categorical inplace.
+        """ Sorts the Categorical by category value returning a new
+        Categorical by default. While an ordering is applied to the
+        category values, sorting in this context refers more to organizing
+        and grouping together based on matching category values. Thus, this
+        function can be called on an unorderd Categorical instance unlike
+        'Categorical.min' and 'Categorical.max'.
 
         Parameters
         ----------
         inplace : boolean, default False
             Do operation in place.
         ascending : boolean, default True
-            Sort ascending. Passing False sorts descending
+            Order ascending. Passing False orders descending. The
```

Comment:
As a native English speaker, I can say that your definitions are perfectly fine :smile: , but when it comes to using the term "sort" in the context of code writing, IINM the connotation is almost always "order".  That's why I logged the issue in the first place.